Biology.java

```diff
@@ -0,0 +1,346 @@
+import cern.colt.Arrays;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * A class that allows antigen to model mutations more realistically, thus, respecting biology
```

Comment:
More realistically than what? Can we be more descriptive here? Like: "A class that allows antigen to model a virus's genetic sequence, and how changes in sequence give rise to changes in both: i) antigenic phenotype and ii) viral fitness in the absence of immune selection (e.g., quantified by DMS data)"